_posts/2017-08-17-community-meeting-update.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: 2017-08-17 Community Meeting Update
+published: true
+author: Jaye Pitzeruse
+company: Indeed
+company-link: https://www.indeed.com
+---
+
+## General Announcements
+
+Call for Papers: CloudNativeCon
+
+CloudNativeCon gathers all CNCF (Cloud Native Computing Foundation) projects under a single roof. 
+Presenters will be talking about their experiences with Kubernetes, Prometheus, OpenTracing, Fluentd, Linkerd, gRPC, CoreDNS, containerd, rkt and CNI.
+The call for papers period will be ending on Monday, August 21, 2017.
+The conference will be taking place on the 6th and 7th of December, 2017.
+If you submit a talk, please add an entry to the spreadsheet linked in the [gRPC Community Meeting Working Doc].
+
+Register for [CloudNativeCon]
+
+## Release Updates
+
+1.6 will be available soon.
+The required changes have already been merged and published in the protocol buffer libraries.
+Keep your eyes pealed for the upcoming release.
+
+## Platform Updates
+
+No platform updates.
+
+## Language Updates
+
+No language specific updates.
+
+[gRPC Community Meeting Working Doc]: https://docs.google.com/document/d/1DTMEbBNmzNbZBh8nOivsnnw3CwUr1Q7WGRe7rNxyHOU/edit#bookmark=id.7qk9qf3ri75m
```

Comment:
Sure can. I like this syntax because it make repeating the link text a bit easier.